main.go

```diff
@@ -27,8 +29,10 @@ func main() {
 
 	config.Load("tigrisdb-cli1", &config.DefaultConfig)
```

Comment:
Do you want to update `tigrisdb-cli1` as well ?

Reply:
Yes :)
In upcoming diff, requires some other unrelated changes.

Reply:
+1